wagtail/images/models.py

```diff
@@ -300,6 +300,10 @@ def _set_image_file_metadata(self):
         self.file.seek(0)
 
     def get_upload_to(self, filename):
+        """
+        generates a file path in the "original_images" folder. Ensuring ASCII characters and limiting length to prevent filesystem issues during uploads.
```

Comment:
```suggestion
        Generates a file path in the "original_images" folder.
        Ensuring ASCII characters and limiting length to prevent filesystem issues during uploads.
```